package.json

```diff
@@ -522,6 +522,12 @@
 					"default": [],
 					"description": "Setting for import paths in your workspace if not using DUB. This will replace other paths. Its recommended to set this in your workspace settings instead of your user settings to keep it separate for each project."
 				},
+				"d.ccdbPath": {
+					"type": "string",
+					"scope": "resource",
+					"default": null,
```

Comment:
wouldn't `compile_commands.json` be a reasonable default?

Reply:
Most (all?) tools that I know of which generate such file work with shadow builds (e.g. CMake, meson).
So the file will likely not be at the project root.
As a user, I find acceptable to manually choose which file to parse for each project, because it also signifies to the tool that I want to use the CCDB to lint the project. Others may disagree with this and I can change to something else (e.g. auto discovery in sub-folder)

Reply:
ah ok, I think it might be best to auto-discover a file, but we can go with this at first (or auto-discover in serve-d eventually, but have this setting to override)

Reply:
I can auto-discover in `serve-d` only if this setting is null?

Reply:
I would recommend doing so yes, otherwise if the user specifies something manually we shouldn't go on searching for something elsewhere.